packages/react/src/Breadcrumbs/__tests__/BreadcrumbsItem.test.tsx

```diff
@@ -20,4 +20,8 @@ describe('Breadcrumbs.Item', () => {
   it('respects the "selected" prop', () => {
     expect(render(<Breadcrumbs.Item selected />)).toMatchSnapshot()
   })
+
+  it('respects the "as" prop', () => {
+    expect(render(<Breadcrumbs.Item as="h2" selected />)).toMatchSnapshot()
+  })
```

Comment:
nit: is this test different than the one in [`behavesAsComponent`](https://github.com/primer/react/blob/main/packages/react/src/utils/testing.tsx#L224) in line 8? (no harm in having it, just curious if there's another purpose)

Reply:
Good call, I missed that. 👍🏻